src/components/study-groups/JoinGroupModal.tsx

```
@@ -0,0 +1,162 @@
+        "Casual Chat": "bg-sky-500/20 text-sky-400 border-sky-500/50",
+        "Exam Cram": "bg-amber-500/20 text-amber-400 border-amber-500/50",
+        "Homework Help": "bg-emerald-500/20 text-emerald-400 border-emerald-500/50",
+    };
+
+    return (
+        <div className="fixed inset-0 z-50 flex items-center justify-center p-4 bg-black/80 backdrop-blur-md">
+            <div className="relative w-full max-w-xl bg-[#0f172a] border border-indigo-500/30 rounded-3xl shadow-2xl flex flex-col overflow-hidden max-h-[90vh]">
+
+                {/* Header Block */}
+                <div className="bg-slate-900 border-b border-slate-800 p-6 relative">
+                    <button onClick={onClose} className="absolute top-6 right-6 p-2 rounded-full bg-slate-800/80 text-slate-400 hover:text-white transition-colors border border-slate-700 hover:bg-slate-700">
```

ESLint

[error] Replace `··········<button·onClick={onClose}·className="absolute·top-6·right-6·p-2·rounded-full·bg-slate-800/80·text-slate-400·hover:text-white·transition-colors·border·border-slate-700·hover:bg-slate-700">` with `<button⏎············onClick={onClose}⏎············className="absolute·top-6·right-6·p-2·rounded-full·bg-slate-800/80·text-slate-400·hover:text-white·transition-colors·border·border-slate-700·hover:bg-slate-700"`

(prettier/prettier)